crates/forge/tests/integration/dispatchers.rs

```diff
@@ -576,7 +576,7 @@ fn nonexistent_method_call() {
     assert_case_output_contains!(
         result,
         "test_nonexistent_method_call",
-        "Entry point selector 0x01fdb214e1495025fa4baf660d34f03c0d8b5037cf10311d2a3202a806aa9485 not found in contract 0x01779e0d4a362d853a9b30a6d6a912ad8a08854d2faff2b0a04e985653d0fb57"
+        "Entry point selector 0x01fdb214e1495025fa4baf660d34f03c0d8b5037cf10311d2a3202a806aa9485 not found in contract 0x00c0727f111a1f9fd7543cf36ae25feaa8184a9188daa4c9c148605445dcba9f"
```

Comment:
Maybe let's remove the address from the assert, since it'll just change between bumps.

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1107